### PR TITLE
[Blog] Using Scoped Traverse to Locate Specific Node

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
@TomasVotruba ready to merge 👍

File: resources/blog/posts/2023/2023-08-21-using-scoped-traverse.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+id: 51
```
Comment (samsonasik) [inline resources/blog/posts/2023/2023-08-21-using-scoped-traverse.md:2]: This needs to be 52 if PR:

- https://github.com/rectorphp/getrector-com/pull/1589

merged first

Comment (TomasVotruba) [reply resources/blog/posts/2023/2023-08-21-using-scoped-traverse.md:2]: No worries, I'l use 52 on the test

File: resources/blog/posts/2023/2023-08-21-using-scoped-traverse.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+public function refactor(Node $node): ?Node
+{
+    if ($node->stmts === null) {
+        return null;
+    }
+
+    // initialize collected returns
+    $returns = [];
+    $this->traverseNodesWithCallable(
```
Comment (samsonasik) [inline resources/blog/posts/2023/2023-08-21-using-scoped-traverse.md:52]: There is `findInstancesOfInFunctionLikeScoped()` already for only this use case, so it need more specific use case

Comment (samsonasik) [reply resources/blog/posts/2023/2023-08-21-using-scoped-traverse.md:83]: I updated to more specific use case https://github.com/rectorphp/getrector-com/pull/1587/commits/4f6a129f28047050d81569ef06d2a8d4cf22ac0d

**Comment by TomasVotruba** [comment]
I'll check this tomorrow and will ship it :+1:  thanks

**Comment by TomasVotruba** [comment]
Ready to go :ship:  Thank you @samsonasik

**Pull request merged**